legacy-docs/formatters.md

```diff
@@ -242,7 +242,7 @@ The JSON structure is like the following example:
 **Machine-parsable**
 
 The `junit` style formatter provides the JUnit formatting.
-This formatter is based on [rubocop-junit-formatter gem](https://github.com/mikian/rubocop-junit-formatter).
+This formatter is based on the [rubocop-junit-formatter gem](https://github.com/mikian/rubocop-junit-formatter).
```

Comment:
You didn't make the same edit to the actual docs. There's no need to touch the markdown files - they will be removed as soon as we setup permanent redirects between the old docs site that used them (powered by RTD) and the new one (docs.rubocop.org).